runtime/lua/vscode-neovim/highlight.lua

```diff
@@ -164,12 +164,9 @@ local function setup()
         -- highlights of custom namespace
         setup_syntax_overrides()
```

Comment:
Uninformed question: is there a reason not to remove this `ColorScheme` conditional as well?

Reply:
Since these two methods have already been executed, there is no need to execute them again unless a theme is set

Reply:
Gotcha. I figured there was no harm in re-executing it (just like with `setup_syntax_groups`), but that make sense. I'm guessing `setup_syntax_groups` is just needed more often than we already execute it?

Reply:
Because setup_syntax_groups cleans up specific syntax highlighting groups, and these highlights are generally defined only after opening a file.